ft_crossfrequencyanalysis.m

```diff
@@ -82,40 +93,46 @@
 freqlow  = ft_checkdata(freqlow,  'datatype', 'freq', 'feedback', 'yes');
 freqhigh = ft_checkdata(freqhigh, 'datatype', 'freq', 'feedback', 'yes');
 
-% prior to 19 Jan 2017 this function had input options cfg.chanlow and cfg.chanhigh,
-% but nevertheless did not support between-channel CFC computations
-cfg = ft_checkconfig(cfg, 'forbidden', {'chanlow', 'chanhigh'});
+if isfield(cfg, 'chanlow') && isfield(cfg, 'chanhigh')
+    docrosschan = true;
+    cfg.chanlow  = ft_channelselection(cfg.chanlow, freqlow.label);
+    cfg.chanhigh  = ft_channelselection(cfg.chanhigh, freqhigh.label);
+    labelcmb = ft_channelcombination({cfg.chanlow,cfg.chanhigh},union(freqlow.label, freqhigh.label));
+    labelcmb(arrayfun(@(x) strcmp(labelcmb{x,:}), 1:size(labelcmb,1)),:) = []; % remove auto
```

Comment:
why?

Reply:
Contrary to what you suggest, I found the `ft_channelcombination` quite useful, however, it seems to add the auto-combination even if I ask not to. That is why I explicitly remove them here.

Reply:
OK, I stand corrected. Apparently ft_channelcombination is smarter than I remember.